lib/gui/app/models/local-settings.js

```diff
@@ -73,15 +77,21 @@ const HOME_CONFIG_PATH = process.platform === 'win32'
  */
 const readConfigFile = (filename) => {
   return new Bluebird((resolve, reject) => {
-    fs.readFile(filename, (error, buffer) => {
+    fs.readFile(filename, { encoding: 'utf8' }, (error, contents) => {
+      let data = {}
       if (error) {
         if (error.code === 'ENOENT') {
-          resolve({})
+          resolve(data)
         } else {
           reject(error)
         }
       } else {
-        resolve(JSON.parse(buffer.toString()))
+        try {
+          data = JSON.parse(contents)
+        } catch (parseError) {
+          // Ignore (?)
```

Comment:
Maybe we should `logDebug` or even `logException` this?

Reply:
@Shou actually same thing as I mentioned below here, as this is also loaded from the main entry point